docs/main/heat-pumps.md

```diff
@@ -44,41 +44,47 @@ More information about the sources behind specifications of heat pump ground can
+_The threshold COP sliders in the ETM can be found in the [Net load](https://energytransitionmodel.com/scenario/flexibility/flexibility_net_load/demand-response-behavior-of-hybrid-heat-pumps) section._
 
-_Checkout: the threshold COP sliders in the ETM in the [Flexibility → Net load](https://pro.energytransitionmodel.com/scenario/flexibility/flexibility_net_load/demand-response-behavior-of-hybrid-heat-pumps) section._
 
-To help you decide the cost-optimal COP setting from a consumer perspective, a special chart is added to the ETM. It shows how much it costs to make a unit of heat with the HHP for space heating. For the gas part, these costs are independent of the COP (and therefore the outside temperature). The costs for the electrical part are decreasing with increasing COP (and increasing outside temperature). The intersection of the two curves is the cost-optimal COP setting for space heating for the given cost price gas and electricity. This assumed cost price of gas and electricity can be set with sliders.
+#### Optimal threshold from a consumer perspective
+To help you decide what the cost-optimal COP setting for space heating is from a consumer perspective, a special chart is available in the ETM. It shows how much it costs to make one unit of heat with the _network gas_ HHP for space heating. For the gas part, these costs are independent of the COP (and therefore the outside temperature). The costs for the electrical part are decreasing with increasing COP (and increasing outside temperature). The intersection of the two curves is the cost-optimal COP setting for space heating for the given cost price gas and electricity. This assumed cost price of gas and electricity can be set with sliders. Note that the monetary unit is euro_cents_.
 
 ![](/img/docs/20210730_HHP_cost_optimal_COP_chart.png)
 
 ![](/img/docs/20200313_HHP_COP_gas_electricity_costs_sliders.png)
 
 The assumptions...
 
-- Gas price 81.4 ct/m3 gas
-- Electricity price 22.5 ct/kWh
+- Gas price 81.4 €ct/m3 gas
+- Electricity price 22.5 €ct/kWh
 - Gas efficiency space heating = 1.07
 
-... lead to a cut off COP for space heating of **2.6**. This cut-off COP is used in the start situation of your scenario. For the HHP on hydrogen a similar cut-off COP is adopted.
+... lead to a cut off COP for space heating of **2.6**. This cut-off COP is used in the starting situation of your scenario. For the HHP on hydrogen a similar cut-off COP is adopted.
 
-If you are interested in the impact of HHPs on the electricity grid, you can have a look at the chart that is shown when clicking on the 'Annual'-button in the right corner of the chart. This chart shows the hourly gas and electricity demand of HHPs.
+#### Optimal threshold from a network perspective
+HHPs have a varying impact on the electricity grid. The chart below shows the hourly gas and electricity demand of HHPs. Different time intervals throughout the year are available under the dropdown menu.
```

Comment:
All in all nice work. Last suggestion is whether we can give users suggestion on how to set the behaviour of HHPs in such a way that the impact on the grid is limited?